lib/tests/test_revset.rs

```diff
@@ -36,9 +37,9 @@ fn test_resolve_symbol_root(use_git: bool) {
     let test_repo = TestRepo::init(use_git);
     let repo = &test_repo.repo;
 
-    assert_eq!(
+    assert_matches!(
         resolve_symbol(repo.as_repo_ref(), "root", None),
-        Ok(vec![repo.store().root_commit_id().clone()])
+        Ok(v) if v == vec![repo.store().root_commit_id().clone()]
     );
```

Comment:
nit: Would using `ok()` on the LHS give us a better message if the assertion fails?

Oh, I see you've used `unwrap()` below. `ok()` both here and below is probably better then?

Reply:
@martinvonz I think using `.ok()` would be worse? In the case that it returns `Err`, you'll immediately want to see more details about the `Err` in question. If we use `.ok()`, the details are thrown away, and you have to go change the code to see that information. `.unwrap()` is annoying in the failure case, but it does preserve the error details. I should probably be more consistent about the usage of `.unwrap()` vs `assert_matches!`, though. Let me know what you think.

Reply:
Yeah, you're probably right that using `assert_matches!` actually gives better error messages. I'm not sure what I was thinking :)